plugins/TodayAtMun/DiaryParser.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+class DiaryParser:
+    DATA_SOURCE = "https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086"
+
+    def __init__(self):
+        self.diary = {}
+        mun_request = requests.get(
+            "https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086", verify=False
```

Comment:
```suggestion
            DATA_SOURCE
```

on my machine, `verify=False` is not required, does this throw an error when you take it away on yours? if so you may wanna look into updating your computers root CAs

also, use the `DATA_SOURCE` here :)

Reply:
and don't forget to run `black` once more after making the changes

Reply:
Ok, should be good now. Thanks, I really appreciate all the critiques!